### PR TITLE
docs: add remote eskip example

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

**Comment by RomanZavodskikh** [comment]
:+1:

File: mkdocs.yml [hunk 1]
```diff
@@ -14,9 +14,10 @@ nav:
         - Video - How to build: tutorials/video-howto-build.md
         - Data Clients:
             - Eskip File: data-clients/eskip-file.md
-            - Etcd: data-clients/etcd.md
-            - Kubernetes: data-clients/kubernetes.md
+            - Remote Eskip: data-clients/eskip-remote.md
             - Route String: data-clients/route-string.md
+            - Kubernetes: data-clients/kubernetes.md
+            - Etcd: data-clients/etcd.md
```
Comment (AlexanderYastrebov) [inline mkdocs.yml:20]: Changed dataclient order from simple to more complex, it also flows naturally when you read doc pages sequentially.

**Comment by szuecs** [comment]
:+1:

File: .gitignore [hunk 2]
```diff
@@ -9,7 +9,6 @@ oauth/user.json
 .idea
 *.iml
 *.xcf
-example.eskip
```
Comment (szuecs) [inline .gitignore:12]: maybe someone has always an example.eskip here

File: mkdocs.yml (hunk 1 above)
Comment (AlexanderYastrebov) [reply mkdocs.yml:20]: Yeah, but eskip file introduces syntax so route string kind of relies on it

File: .gitignore (hunk 2 above)
Comment (AlexanderYastrebov) [reply .gitignore:12]: we'll see :)

**Pull request merged**